DOCS: add help for quicksetup fails in intro (#4838)
aiidateam/aiida-core: docs/source/intro/install_conda.rst

       Last name: name
       Institution: where-i-work

   .. admonition:: \ \

>        :class: attention title-icon-troubleshoot

Reviewer: Ok, sorry, I did not intend to motivate the introduction of some work-around, I just did not know whether this kind of admonition requires a title or not. Let's stick with the standard style with title then.

       Last name: name
       Institution: where-i-work

>    .. admonition:: Having problems auto-detecting the PostgreSQL setup?

Reviewer: Do we really need a title here? I think the current one also sounds a bit weird since it would be AiiDA having issues with auto-detecting the setup, not the user.

       :class: attention title-icon-troubleshoot

>        Sometimes the local configuration of postgres may be difficult to auto-detect for AiiDA.

Reviewer: You capitalized postgres in the title. If we do it there, we should it everywhere.

>        If that is your case, you can try to do the process manually by first :ref:`creating the database yourself<intro:install:database>` and then :ref:`running the full setup command<intro:install:verdi_setup>`.

Reviewer: ```suggestion
       In that case try to  :ref:`create the database manually<intro:install:database>` and then :ref:`run the full setup command<intro:install:verdi_setup>`.
```

       Last name: name
       Institution: where-i-work

>    If AiiDA has any trouble autodetecting the PostgreSQL setup, you can check the optional manual setup explained in the :ref:`troubleshooting section<intro:troubleshooting:installation:quicksetup>`.

Reviewer: I think this was a misunderstanding. I did not mean to completely remove the admonition container, just its title (if possible).

Author: So, I found [a way](https://stackoverflow.com/questions/61435028/how-to-create-boxed-text-without-title-in-sphinx) to do it, but the box does not look exactly the same (the orange line where the title would be is thinner and the symbol escapes from it). Let me know if you think it is good or if I should revert to a title.
